eudemo/eudemo/power_cycle.py

```diff
@@ -172,11 +181,12 @@ def run(self, run_result):
         }
 
 
-class SteadyStatePowerCycleSolver(SolverABC):
+class SteadyStatePowerCycleSolver(CodesSolver):
```

Comment:
Did I miss this when removing the `SolverABC`? Whoops!

This isn't really an external code solver, do we need to open an issue to revisit this at some point?

Reply:
Yeah I'll open an issue

Reply:
#1923